server/kit/kitserver.go

```diff
@@ -87,12 +88,31 @@ func NewServer(svc Service) *Server {
 		errs  *errorreporting.Client
 		propr propagation.HTTPFormat
 	)
-	// if in App Engine, initiate an error reporter and the stackdriver exporters
-	if isGAE() {
-		projectID, serviceID, svcVersion := getGAEInfo()
+
+	projectID := googleProjectID()
+	if projectID != "" {
```

Comment:
While developing locally, some folks may have this variable set so they can interact with remote resources (like Spanner, that doesnt have an emulator).

Because of that, we may not want to make that variable the gate for turning on error reporting. Can we alter things below a tad so that the `errorreporting` client only gets initiated if we also have initiated the Stackdriver exporter?

Reply:
dibe